tests/test_transforms.py

```diff
@@ -188,7 +188,7 @@ def test_transformed_logprob(at_dist, dist_params, sp_dist, size):
             )
 
         exp_logprob_val += np.log(np.linalg.det(jacobian_val))
-        exp_logprob_val += b_dist.logpdf(b_val)
+        exp_logprob_val += b_dist.logpdf(b_val).sum()
```

Comment:
Here we had an incorrect logprob in the `dirichlet` test, because the normal `b_dist (size=2)` was broadcasting the otherwise scalar `a_dist.logpdf(a_val) (size=())`